### DISCUSSION TITLE
What's that?

### DISCUSSION THREAD

**Discussion by euronymous** [tchiotludo/akhq#1011] [General]
I wanted to connect akhq to a production kafka instance, so I've checked first what's inside the docker compose.

https://github.com/tchiotludo/akhq/blob/a5eb393f09cd2b4f0e7f034a149f1bbb52b6e162/docker-compose.yml#L120

**Comment by tchiotludo** [comment]
Just a simple injector to start a cluster with some data (for the demo only)
